### PR TITLE
[ i18n sprint ] Move translations from language specific lib-home-page templates to properties.

### PR DESCRIPTION
**Pull request by litvinovg**
[Companion PR](https://github.com/vivo-project/VIVO/pull/3780)
#What does this pull request do?#
Moves translations from language specific lib-home-page templates to properties.
#How to test#
Open home page, view list of research types

### CODE REVIEW

**Review by bkampe — COMMENTED** [review]
German translations are fine

File: es/webapp/src/main/webapp/i18n/vivo_all_es.properties [hunk 1]
```diff
@@ -214,6 +214,8 @@ define_value_for_property = Para utilizar esta función, defina un valor para es
 faculty_capitalized = Facultad
```
Comment (AnnaGuillaumet) [inline es/webapp/src/main/webapp/i18n/vivo_all_es.properties]: Profesorado

File: es/webapp/src/main/webapp/i18n/vivo_all_es.properties [hunk 2]
```diff
@@ -214,6 +214,8 @@ define_value_for_property = Para utilizar esta función, defina un valor para es
...
 loading_faculty = Cargando facultad. . .
```
Comment (AnnaGuillaumet) [inline es/webapp/src/main/webapp/i18n/vivo_all_es.properties]: Cargando profesorado...

File: es/webapp/src/main/webapp/i18n/vivo_all_es.properties [hunk 3]
```diff
@@ -214,6 +214,8 @@ define_value_for_property = Para utilizar esta función, defina un valor para es
...
 research_capitalized = Investigación
+items_of_type = Objetos de tipo
+thousands_short = k
 view_all = Ver todo ...
 view_all_research = Ver todas las investigaciones
 no_research_content_found = No se encontró contenido de investigación.
```
Comment (AnnaGuillaumet) [inline es/webapp/src/main/webapp/i18n/vivo_all_es.properties]: No se ha encontrado contenido de investigación.

**Review by AnnaGuillaumet — COMMENTED** [review]
Is that all?

File: es/webapp/src/main/webapp/i18n/vivo_all_es.properties (hunk 1 above)
Comment (AnnaGuillaumet) [inline es/webapp/src/main/webapp/i18n/vivo_all_es.properties]: Profesorado

**Review by chenejac — APPROVED** [review]
It works for me. 
@litvinovg please adopt Anna's corrections for Spanish.

File: es/webapp/src/main/webapp/i18n/vivo_all_es.properties (hunk 1 above)
Comment (litvinovg) [reply es/webapp/src/main/webapp/i18n/vivo_all_es.properties]: Fixed. Thank you.

File: es/webapp/src/main/webapp/i18n/vivo_all_es.properties (hunk 2 above)
Comment (litvinovg) [reply es/webapp/src/main/webapp/i18n/vivo_all_es.properties]: Fixed. Thank you.

File: es/webapp/src/main/webapp/i18n/vivo_all_es.properties (hunk 3 above)
Comment (litvinovg) [reply es/webapp/src/main/webapp/i18n/vivo_all_es.properties]: Fixed. Thank you.

**Review by bkampe — APPROVED** [review]

**Review by bkampe — APPROVED** [review]

**Review by chenejac — APPROVED** [review]

**Pull request merged**